tests/e2e_logging.py

```diff
@@ -1630,23 +1634,36 @@ def test_committed_index(network, args):
 
     _, log_id = network.txs.get_log_id(txid)
 
-    r = network.txs.request(log_id, priv=True, url_suffix="committed")
+    remaining_retries = 10
```

Comment:
That retry logic would be better places in network.txs.request(..., timeout=...) I think. Parsing the error message seems unnecessarily brittle too.

Reply:
That makes sense, and I agree parsing the error message is a bad approach I'm trying to get something working that I can then clean up

Reply:
Change now lives in #4803